scripts/process_systems.py

```diff
@@ -1,216 +1,526 @@
-                                    "<span class=\"fas fa-user-cog color-1\"></span></div></div>" \
-                                    "<div class=\"media-body p-2 mt-3\"><h3 class=\"heading\">Tutorial</h3></div></div></a>" if "tutorial" in data["documentation"] else ""
-
-        # execution environment
-        if "execution_environment" in data:
-            s["execution_environment"] = "<div class=\"row justify-content-center mt-5\">" \
-                                         "<div class=\"col-md-7 text-center heading-section ftco-animate\">" \
-                                         "<h3 class=\"mb-4\">Execution Environment</h3></div></div>" \
-                                         "<div class=\"row\">"
-
-            # user interfaces
-            if "interfaces" in data["execution_environment"]:
-                s["execution_environment"] += "<div class=\"col-md-4 ftco-animate\">" \
-                                              "<ul class=\"list-services\">" \
-                                              "<li><h4>User Interfaces</h4></li>"
-                for i in data["execution_environment"]["interfaces"]:
-                    s["execution_environment"] += f"<li>{i}</li>"
-                s["execution_environment"] += "</ul></div>"
-
-            # resource managers
-            if "resource_managers" in data["execution_environment"]:
-                s["execution_environment"] += "<div class=\"col-md-4 ftco-animate\">" \
-                                              "<ul class=\"list-services\">" \
-                                              "<li><h4>Resource Managers</h4></li>"
-                for rm in data["execution_environment"]["resource_managers"]:
-                    s["execution_environment"] += f"<li>{rm}</li>"
-                s["execution_environment"] += "</ul></div>"
-
-            # transfer protocols
-            if "transfer_protocols" in data["execution_environment"]:
-                s["execution_environment"] += "<div class=\"col-md-4 ftco-animate\">" \
-                                              "<ul class=\"list-services\">" \
-                                              "<li><h4>Transfer Protocols</h4></li>"
-                for tp in data["execution_environment"]["transfer_protocols"]:
-                    s["execution_environment"] += f"<li>{tp}</li>"
-                s["execution_environment"] += "</ul></div>"
-            
-            s["execution_environment"] += "</div>"
-
-    # release component
-    if s["release_name"]:
-        if s["release_url"]:
-            s["release"] = f"<a href=\"{s['release_url']}\" target=\"_blank\"><strong>{s['release_name']}</strong> " \
-                f"</a> &mdash; Released on: {s['release_date']}"
+        metadata = yaml.safe_load(r.text)
+        _apply_wci_metadata(metadata, model)
+
+    return model
+
+
+def _process_gitlab_system(definition: GitLabDefinition) -> SystemMetadata:
+    acces_token = os.environ.get("GITLAB_TOKEN")
```

Comment:
For the gitlab support to work a GITLAB_TOKEN environment variable is needed, similar to JEKYLL_TOKEN.